src/docfx/restore/PackageResolver.cs

```diff
@@ -128,7 +129,7 @@ private PathString DownloadGitRepositoryCore(string url, string committish, bool
             return gitPath;
         }
 
-        private void InitFetchCheckoutGitRepository(string cwd, string url, string committish, bool depthOne)
+        private void InitFetchCheckoutGitRepository(string cwd, string url, string committish, bool depthOne, bool ignoreBranchFallback)
```

Comment:
nit: consider passing the flags here
```suggestion
        private void InitFetchCheckoutGitRepository(string cwd, string url, string committish, PackageFetchOptions flags)
```